pkg/templates/charts/toggle/cluster-proxy-addon/templates/user-deployment.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     component: cluster-proxy-addon-user
     chart: cluster-proxy-addon-2.1.0
 spec:
-  # fixed replicas to 1, the kubectl-ui does not support multiple replicas
+  # fixed replicas to 1, the user-server does not support multiple replicas
```

Comment:
This caught my eye. This says the deployment must be fixed to 1 replica yet below the replica count is configurable.  @xuezhaojun will this cause problems?

Reply:
It won't, I think it's a outdate comments, I will fix later.